src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const DAILY_FEE = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let sum = days * DAILY_FEE;
```

Comment:
Consider using `const` instead of `let` for `sum` as it is not reassigned after its initial declaration. This helps to prevent unintentional reassignments and makes the code more robust.